pages/exporthardcopy.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Export Hardcopy</title>
+</head>
+<body class="small_foot_body">
+<div id="loading" style="display:none" class="hidden-print">
+  <div id="loadmiddle">
+    <div id="loadinner">
+      <p>Loading<span class="dots"><span>.</span><span>.</span><span>.</span></span></p>
+      <div class="loadprogress" role="button" aria-label="progress"></div>
+    </div>
+  </div>
+</div>
+<div id="form" class="hidden-print" style="display:none">
+  <div>
+    <form role="form" class="form-horizontal" onsubmit="return false">
+      <fieldset>
+        <legend>Export Hardcopy</legend>
+        <div class="form-group">
+          <label class="col-md-4 control-label" for="tasker">Tasking Unit/Operations Centre</label>  
+          <div class="col-md-4">
+            <input id="tasker" name="tasker" type="text" placeholder="Heresville SES" class="form-control input-md" required="">            
+          </div>
+        </div>
+
+        <!-- Text input-->
+        <div class="form-group">
+          <label class="col-md-4 control-label" for="tasker_email">SES Contact Email</label>  
+          <div class="col-md-4">
+            <input id="tasker_email" name="tasker_email" type="text" placeholder="eg unit.ops@ses.nsw.gov.au" class="form-control input-md">
+            <span class="help-block">Ensure this email address is monitored</span>
+          </div>
+        </div>
+
+        <!-- Text input-->
+        <div class="form-group">
+          <label class="col-md-4 control-label" for="tasker_phone">SES Contact Phone</label>  
+          <div class="col-md-4">
+            <input id="tasker_phone" name="tasker_phone" type="text" placeholder="eg 0242516111" class="form-control input-md">
+            <span class="help-block">Ensure this phone number is contactable</span>  
+          </div>
+        </div>
+        <div class="form-group">
+          <label class="col-md-4 control-label" for="tasker_fax">SES Contact Fax</label>  
+          <div class="col-md-4">
+            <input id="tasker_fax" name="tasker_fax" type="text" placeholder="eg 0242516222" class="form-control input-md">
+            <span class="help-block">(Optional) Ensure this fax number is monitored</span>
+          </div>
+        </div>
+        <!-- Button (Double) -->
+        <div class="form-group">
+          <div class="col-md-8">
+            <button id="goButton" name="goButton" class="btn btn-success">Create Jobsheets for Print/Fax</button>
```

Comment:
I was looking to allow the User to enter contact details for the Op Centre making the printout/handling the tasking. Still considering using it, so just hid it for the time being.